spark/v4.0/spark/src/main/java/org/apache/iceberg/spark/SparkConfParser.java

```diff
@@ -40,7 +40,7 @@ class SparkConfParser {
 
   SparkConfParser() {
     this.properties = ImmutableMap.of();
-    this.sessionConf = new RuntimeConfig(SQLConf.get());
+    this.sessionConf = new org.apache.spark.sql.classic.RuntimeConfig(SQLConf.get());
```

Comment:
Why do we have both sql.RuntimeConfig and sql.classic.RuntimeConfig ?

Reply:
I tried to use `sql.RuntimeConfig` as much as possible and only switch to `sql.classic.RuntimeConfig` when necessary. That's why I have both `sql.RuntimeConfig` and `sql.classic.RuntimeConfig`.